scanner/src/test/kotlin/ScanCodeTest.kt

```diff
@@ -43,6 +43,9 @@ class ScanCodeTest : WordSpec({
         "be correctly summarized" {
             val resultFile = File("src/test/assets/mime-types-2.1.18_scancode-2.9.7.json")
             val result = scanner.getRawResult(resultFile)
+
+            // Misuse the "resultFile" as the "scanPath" even if semantical nonsense to have a simpe way for testing
```

Comment:
Typo: "simple", same in comments below.
I don't see a need for the harsh wording, also I think it's good to explain why this is semantically wrong. What about this sentence instead?
`The "scanPath" argument should point to the scanned files, but since they are not available here pass the "resultFile" instead to test the calculation of the package verification code.`

Reply:
I've reworded the sentence based on your suggestion.